src/cartesian/Line.tsx

```diff
@@ -272,16 +272,24 @@ export class Line extends PureComponent<Props, State> {
       };
     }
 
-    return errorBarItems.map((item: ReactElement<ErrorBarProps>, i: number) =>
-      React.cloneElement(item, {
-        // eslint-disable-next-line react/no-array-index-key
-        key: `bar-${i}`,
-        data: points,
-        xAxis,
-        yAxis,
-        layout,
-        dataPointFormatter,
-      }),
+    const errorBarProps = {
+      clipPath: needClip ? `url(#clipPath-${clipPathId})` : null,
```

Comment:
Lets be as specific as possible.
```suggestion
      clipPath: needClip===true ? `url(#clipPath-${clipPathId})` : null,
```

Reply:
I just saw that this line is reimplemented everywhere in the code. So good to go (i.e. for the sake of consistency).